CHANGELOG.md

```diff
@@ -0,0 +1,99 @@
+- Reject non-integer Duration fields in Duration.with() ([e6b2488d], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1735))
+- Ensure an Object is returned from calendar.mergeFields() ([4e63f25f], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1719))
+
+Bug fixes:
+
+- Fix GetFormatterParts for Firefox Nightly ([47f9132f])
+- Fix TS types of RoundTo and TotalOf ([3008a670])
+- Fix crash setting `day` outside current JPN era ([6d3588c3], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1807))
+- Copy options object for PlainYearMonth.{add,subtract} and InterpretTemporalDateTimeFields to prevent user-modified objects from interfering with later operations. ([bafa1bdf], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1748))
+- Validate input to Calendar.prototype.fields ([7ebc700e], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1750))
+- Stop observably calling into `getPossibleInstantsFor` in `InterpretISODateTimeOffset` ([5448e59f], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1688))
+- Call `CalendarEquals` correctly ([07ea694e], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1858))
+- Fix arithmetic issues when using non-ISO months ([079a3325], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1761))
+
+Non-breaking changes:
+
+- Various `#round` and `#total` methods now accept string parameters or options bags. Strings are interpreted as the `smallestUnit` option (or `unit` for `Temporal.Duration#total`). ([068e801f], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1875))
+- Add @@toStringTag to TS types ([41ab6bc0])
+- Accept string Calendar names in PlainMonthDay and PlainYearMonth constructors ([27b4c7e8])
+- Make options optional in Calendar method TS types ([3a09d00d])
+- Regex: tighten matching of month and day values in datesplit ([b5736546], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1836))
+
+Other:
+
+- Bump various dependencies ([47701107], [f5427de9], [310d9d8b])
+- Allow launching and debugging tests (both Demitasse and Test262 suites) from the VSCode debug panel ([960d9b76], [7f7c19a1], [4ec6568e], [edcc668b])
+- Run the Test262 test suite against this polyfill, for various configurations of the resulting build artifact ([2331468d], [666c69da], [429273ec], [ff937782], [f885253f])
+- Remove various pieces of unused code, and add CI testing to detect unused code in PR review ([67f9f6bb], [63bdfcd1])
+- Drop the dependency on es-abstract ([d24575f2], [ad7e2e3a], [5b1bc5e2])
+- The polyfill's source was ported to TypeScript ([12e4d529], [ac78fd9d], [53f32e0f], [06b806c9], [66fdc765], [50b1c34b], [4724b017], [947a8a5e], [fdbf7e01], [fa60af6a], [da753f2f], [f4db8b0b], [4a38420d])
+- Document the release process for this polyfill ([c55818b6])
```

Comment:
This is so cool. Thanks for doing this!

Reply:
Of course! I hope to automate this process just a little more in the future so we can always be certain this changelog makes sense.

Something that would help greatly I think going forward would be some sort of check during PR review to either tag a PR with the relevant spec PRs, or to explicitly acknowledge that there are no corresponding PRs. I'm going to ponder this a bit over the holidays and might throw something together for us to consider in the new year.